include/sys/sysevent/dev.h

```diff
@@ -244,6 +244,8 @@ extern "C" {
 #define	DEV_PATH		"path"
 #define	DEV_IS_PART		"is_slice"
 #define	DEV_SIZE		"dev_size"
+#define	DEV_LINKS		"dev_links"
+#define	DEV_IS_MPATH		"is_mpath"	/* Is multipath */
```

Comment:
If "mpath" is not clear enough, and needs a comment, maybe s/is_mpath/is_multipath/ through the whole patch?